druid-shell/src/platform/gtk/window.rs

```diff
@@ -479,7 +499,7 @@ impl WindowHandle {
     /// Bring this window to the front of the window stack and give it focus.
     pub fn bring_to_front_and_focus(&self) {
         //FIXME: implementation goes here
-        log::warn!("bring_to_front_and_focus not yet implemented for gtk");
+        warn!("bring_to_front_and_focus not yet implemented for gtk");
```

Comment:
I think @cmyr wants us to not use imported functions directly. That is `log::warn!` is better than `warn!`. It's from the [Rust style guide](https://doc.rust-lang.org/1.0.0/style/style/imports.html#prefer-fully-importing-types/traits-while-module-qualifying-functions).

Reply:
I prefer that too, was just done differently in the Windows version I used as reference.
Wonder if there are lints for such things :thinking:

Reply:
Yeah the windows platform code has some of the oldest code in druid and so follows style the least.

I've been also thinking about looking into custom clippy/rustfmt options to enforce some of our style rules.

Reply:
yea, this is a nit but I like macros and functions to be module-qualified. Would gladly accept a PR that standardizes on this :)